scripts/travis/kafka-integration-test.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -e
+
+docker pull spotify/kafka
```

Comment:
Maybe pin the version? What if the image is upgraded and our driver is not compatible.